src/types.ts

```diff
@@ -590,10 +590,14 @@ export type UserResponse<UserType = UnknownType> = User<UserType> & {
  */
 
 export type BanUserOptions<UserType = UnknownType> = UnBanUserOptions & {
+  banned_by_id?: string;
```

Comment:
didn't the backend changes also include `BannedBy`?

Reply:
Yes. I'm not exactly sure how it could be used from the SDK as we don't had it previously. I was following Tommaso's suggestion to support everything we had previously, including `User` field. So old `User` became `BannedBy`

Reply:
there's a `user` param here on line 597

Reply:
I'm going to add some tests cases for this one, will check if it even works